config/credentials.yml.enc

```diff
@@ -1 +1 @@
-NZBwq9bjgOQK3UcnU+YfExAAyPcOI5R2TjX5aaO1FWnYiROncJh+ztnfiYqFIsBz0Q1R3N8f7cmGqj1nxmYUEbgvGNh43ShqKj5FvgudK6sbOjM7W/l2jKJgDjpGgG7RLFOLf3RRyBkuVjnYWkftHQbur7ceRzJ1Xy+wnk5deHmRXAm0goYx3v8cX4UEUp2L+oLX1dqI8O/ss2PFYCC8NmpQB18Z3AMxfBmk9xdyY6CQW5YYzUJLFfy/leVxn0eJ/SCxcEdUYLlcw3u2D7sjTn1QHE49vHQ8J6HzZH1ZSAcBkX0PkxryWw1yifJyvon3ulvLQb8tOlCArE0v71ZwLNJdmB6E0knhecFfFOHYHZO+v7jsDFdr82ZbAtJp9SM/70uMAEmh8Gv4Xa44zdcxoDdFlPaji6GeEad5--0ugWqKp5QkeBPXQN--bT0Xiem7FRbnoFtU0qlbpg==
\ No newline at end of file
+/KanGLXNZi3YrwW4KNXWORH67F/Db1hJDEJuQN9CQJcQl1xm96wLTZFhw8bATw7//JeQR+lJ81ESh33pojjpNFm4vjzsMAMaHWkNxc3RjrEk9u+n4TcSHqWAdf7YPAvLGT+anuRo3CC9zTI1sZ1YW+tAnaq63RPKVkjpcLeHsIxbXXX5UjfHZRKEuZ+zLqfB7GweMw4HpW40lNvVWSEAXxxCP5ouz2cwlNULZEWiNDrJDjW/arbds7CuOClXE0Rc89SbEaiV/eIoB4rRA9IuJxj8FyTVBfTn4IaHhiLe+RDxUzQKex2V20TZZbRHGeN5FjXuaMZu2RNjkhixksK5R0EF8YSD60lh4b2AwPnAnmfzq42onYiGtGKBSNKPQ94fK+QicVNcK8iLbabkcwayNxF3qzKS5Bm8k/C7o8rKOROktENQFYoYn6X1pR0UZ61h1ydReSB5VbsbM4uyKkCDcvK0yfs7A+53K70KecL09oBYq05a5EQLcfyNnu7PlTBLOY0uUyrYPlPy+vK/z4ZyMYuHy2oGXWaQYfMHmzsyw7vko5uaqHJ8Oc9oD4IZWL9q2IW4gkso065CyIfe3s2a3u+vQnPF+NDOfmiwgepG4eHZky4tY1scYI2kgXg=--m4aamBCyhk2WMYZh--D1IYu/gkTyQRGtFeG1sOXg==
```

Comment:
Please add last empty line.

Reply:
This file itself is encrypted, I can’t add